File: docs/examples/record_api_dotnet/List.cs
using TrailBase;
using System.Text.Json.Nodes;

public partial class Examples {
  public static async Task<ListResponse<JsonObject>> List(Client client) =>
    await client.Records("movies").List(
        pagination: new Pagination(limit: 3),
        order: ["rank"],
        filters: [
          new Filter(column:"watch_time", op:CompareOp.LessThan, value:"120"),
          new Filter(column:"description", op:CompareOp.Like, value:"%love%"),
        ]);
}

File: docs/examples/record_api_py/record_api_py/list.py
from trailbase import Client, Filter, CompareOp, JSON_OBJECT


def list_movies(client: Client) -> list[JSON_OBJECT]:
    response = client.records("movies").list(
        limit=3,
        order=["rank"],
        filters=[
            Filter(column="watch_time", value="120", op=CompareOp.LESS_THAN),
            Filter(column="description", value="%love%", op=CompareOp.LIKE),
        ],
    )

    return response.records
